src/Console/ReplayCommand.php

```diff
@@ -70,7 +64,9 @@ public function selectProjectors(array $projectorClassNames): ?Collection
 
     public function replay(Collection $projectors, int $startingFrom): void
     {
-        $replayCount = $this->getStoredEventClass()::startingFrom($startingFrom)->count();
+        $repository = app(StoredEventRepository::class);
+        $events = $repository->retrieveAll(null, $startingFrom);
```

Comment:
The interface expects to have a Collection returned? Imagine we have millions of events. This would load them all in memory right? Can this be refactored so we can retrieved them in a chunked way?

Reply:
Since this PR targets Laravel 6, could we do something with this? https://github.com/laravel/framework/pull/29415

#cuttingedge 🔪

Reply:
I like that idea very very much 🔥

Reply:
Updated to use the new LazyCollection!